code/game/machinery/vending.dm

```diff
@@ -996,9 +996,12 @@
 					/obj/item/weapon/gun/projectile/olivaw = 5,
 					/obj/item/weapon/gun/projectile/giskard = 5,
 					/obj/item/weapon/gun/energy/gun/martin = 5,
+					/obj/item/weapon/gun/energy/gun = 3
```

Comment:
bruh

Reply:
fuck i forgot the comma

Reply:
you fucking put a parent into the list
go test it or something

Reply:
parent is the spider rose for some reason and martin and mounted are variations on that

Reply:
bruh

Reply:
> you fucking put a parent into the list
> go test it or something

![shitcode](https://user-images.githubusercontent.com/39888997/125699088-890be37a-0435-4830-893c-9ffeba92c9a4.png)
i have tested it and it works fine